subprojects/studio-plugin/src/main/java/org/gradle/profiler/studio/plugin/GradleProfilerProjectManagerListener.java

```diff
@@ -25,6 +26,9 @@ public class GradleProfilerProjectManagerListener implements ProjectManagerListe
     public void projectOpened(@NotNull com.intellij.openapi.project.Project project) {
         LOG.info("Project opened");
         if (System.getProperty(PROFILER_PORT_PROPERTY) != null) {
+            // Skip Android Studio Gradle automatic import on project open,
+            // interestingly this is an issue only with Android Studio but not IntelliJ
+            GradleProjectInfo.getInstance(project).setSkipStartupActivity(true);
```

Comment:
How does this affect benchmarks? Are we doing the import during the first warmup round perhaps, or not at all?

Reply:
For context: this solves the issue where Android Studio would run the Gradle sync automatically on first import. Unfortunately it seems we can't always detect it. And when we trigger a sync it could happen that two Gradle connections would be started: one for automatic sync and one for our sync. That resulted that two syncs were run simultaneously.

With this line we disable that automatic sync, but we still trigger a sync on first warmup. It's just that this workarounds an issue with possible two simultaneously syncs on that first warmup import.

Reply:
Could you add this explanation in the code please?